libs/sdk-backend-base/src/cachingBackend/index.ts

```diff
@@ -380,8 +455,52 @@ export type CachingConfiguration = {
      * @param cacheControl - cache control instance
      */
     onCacheReady?: (cacheControl: CacheControl) => void;
+
+    /**
+     * Maximum number of organizations that will have its security settings cached. The scope string built by
+     * particular backend implementation from organization ID will be used as cache key.
+     *
+     * When limit is reached, cache entries will be evicted using LRU policy.
+     *
+     * When no maximum number is specified, the cache will be unbounded and no evictions will happen. Unbounded
+     * security settings organization cache is dangerous in applications with long-lived sessions that can
+     * create many unique requests and memory usage will only go up.
+     *
+     * When non-positive number is specified, then no caching will be done.
+     */
+    maxSecuritySettingsOrganizations: number | undefined;
```

Comment:
small stuff: i wonder if we can find some shorter name for this.. not other words but maybe shortening `Organizations` to just `Orgs`

Reply:
Fixed. I went with "Org(s)". I also thought that it is too long but was not sure how to shorten it. I don't like shortened words usually but it is better now.